shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/constant/TransactionIsolationLevel.java

```diff
@@ -28,9 +28,9 @@
 public enum TransactionIsolationLevel {
 
     NONE("NONE"),
-    READ_UNCOMMITTED("READ_UNCOMMITTED"),
```

Comment:
Are these changes related to the issue?

Reply:
Keep it the same as the real database value